notebooks/exemples/1 - Simple exemple - MNIST.ipynb

```diff
@@ -0,0 +1,1217 @@
+{
```

Comment:
It's exAmples in English, would you please rename the folder?

Reply:
Yes, thank you